Dockerfile

```diff
@@ -0,0 +1,28 @@
+# Compile
+FROM    alpine:3.10 AS compiler
+
+RUN     apk update --quiet
+RUN     apk add curl
+RUN     apk add build-base
+RUN     apk add libressl-dev
+
+RUN     curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- -y
+
+WORKDIR /meilisearch
+
+COPY    . .
+
+ENV     RUSTFLAGS="-C target-feature=-crt-static"
+
+RUN     $HOME/.cargo/bin/cargo build --release
+
+# Run
+FROM    alpine:3.10
+
+RUN     apk update --quiet
+RUN     apk add libressl
+RUN     apk add build-base
+
+COPY    --from=compiler /meilisearch/target/release/meilidb-http .
+
+CMD     ./meilidb-http
```

Comment:
Did you try to run that? I remember that alpine isn't shipped with a libc and that was a pain in the ass to make Rust work without it and use musl instead. I would prefer using Debian.

Reply:
This was not about the docker image because we are not using the docker image for the test. It was the GitHub actions that was not using the latest rust version, and I just fixed it :)

Reply:
Yes of course, that's just the Github Review interface that make it like those two comments are related :)